### DISCUSSION TITLE
A few question

### DISCUSSION THREAD

**Discussion by PrzemoCe** [OpenOrienteering/mapper#2297] [Questions & Answers]
Hello guys,
I have two questions regarding the Mapper:

1. How can I export a map with a transparent background? I cannot disable the default white one.
2. How can I enable the vector graphics mode while exporting a map? For me it's always greyed out.
![Capture](https://github.com/user-attachments/assets/0a216444-1990-4095-89bc-b5619c574ae9)

Thanks in advance for your help :)

**Comment by dl3sdo** [comment]
Regarding your 2nd question:
I assume that you are using version 0.95. This version has a known issue with vector graphics export (see #1914).
Either use a current build (e.g., the one from today, those builds are as stable as any release version, howto in #1914) or use version 0.94 for exporting.

**Comment by dl3sdo** [comment]
Hello @PrzemoCe,

1. The export of a map with transparent background is already in the master branch and thus part of next Version 0.9.7. See #2442.
2. The current version [0.9.6](https://github.com/OpenOrienteering/mapper/releases/tag/v0.9.6) solves this issue.
